x-pack/plugins/apm/server/routes/transactions/transactions_routes.ts

```diff
@@ -4,21 +4,26 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import * as t from 'io-ts';
 import Boom from '@hapi/boom';
-import { setupRequest } from '../lib/helpers/setup_request';
-import { getTransactionCharts } from '../lib/transactions/charts';
-import { getTransactionDistribution } from '../lib/transactions/distribution';
-import { getTransactionBreakdown } from '../lib/transactions/breakdown';
-import { getTransactionGroupList } from '../lib/transaction_groups';
-import { createRoute } from './create_route';
-import { uiFiltersRt, rangeRt } from './default_api_types';
-import { getTransactionSampleForGroup } from '../lib/transaction_groups/get_transaction_sample_for_group';
-import { getSearchAggregatedTransactions } from '../lib/helpers/aggregated_transactions';
-import { getErrorRate } from '../lib/transaction_groups/get_error_rate';
-
-export const transactionGroupsRoute = createRoute({
-  endpoint: 'GET /api/apm/services/{serviceName}/transaction_groups',
+import * as t from 'io-ts';
+import { toNumberRt } from '../../../common/runtime_types/to_number_rt';
+import { getSearchAggregatedTransactions } from '../../lib/helpers/aggregated_transactions';
+import { setupRequest } from '../../lib/helpers/setup_request';
+import { getServiceTransactionGroups } from '../../lib/services/get_service_transaction_groups';
+import { getTransactionBreakdown } from '../../lib/transactions/breakdown';
+import { getTransactionCharts } from '../../lib/transactions/charts';
+import { getTransactionDistribution } from '../../lib/transactions/distribution';
+import { getTransactionGroupList } from '../../lib/transaction_groups';
+import { getErrorRate } from '../../lib/transaction_groups/get_error_rate';
+import { createRoute } from '../create_route';
+import { rangeRt, uiFiltersRt } from '../default_api_types';
+
+/**
+ * Returns a list of transactions grouped by name
+ * //TODO: delete this once we moved away from the old table in the transaction overview page. It should be replaced by /transactions/groups/overview/
```

Comment:
Thanks for adding a todo and a helpful comment. 👍 

I think in general, more of our APIs and helper functions could use comments